src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -442,6 +576,64 @@ const FormCreationForm = ({csrftoken, formUuid, formHistoryUrl }) => {
         }
     };
 
+    const onRuleChange = (index, event) => {
+        const { name, value } = event.target;
+        dispatch({
+            type: 'CHANGED_RULE',
+            payload: {name, value, index},
+        });
+    };
+
+    const onRuleDelete = (index) => {
+        dispatch({
+            type: 'DELETED_RULE',
+            payload: {index: index}
+        });
+    };
+
+    const onRuleAdd = () => {
+        dispatch({type: 'ADD_RULE'});
+    };
```

Comment:
You can simplify this and inline it in the prop where it's used rather than defining another local function. That reduces visual clutter a bit.